### PR TITLE
Use Helsinki Open Data WFS layers for feature map

### PR DESCRIPTION
**Pull request by mingfeng**
Screenshot from 2020-10-22 11-28-19

Refs: #283

### CODE REVIEW

File: ltj/settings.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ def get_git_revision_hash():
     LOG_LEVEL=(str, "INFO"),
     SENTRY_DSN=(str, ""),
     SENTRY_ENVIRONMENT=(str, "unconfigured"),
-    WFS_SERVER_URL=(str, "http://localhost/"),
-    WFS_NAMESPACE=(str, "ltj-dev"),
+    WFS_SERVER_URL=(str, " https://kartta.hel.fi/ws/geoserver/avoindata/wfs"),
```
Comment (frwickst) [inline ltj/settings.py:47]: Is it just me or is there a space between the `"` and the `h`

**Review by frwickst — CHANGES_REQUESTED** [review]
Space, the final frontier! 👾

File: ltj/settings.py (hunk 1 above)
Comment (mingfeng) [reply ltj/settings.py]: Fixed now

**Review by frwickst — APPROVED** [review]
🚢 🚢 🚢 🚢

**Pull request merged**